pkg/runtime/controller.go

```diff
@@ -84,10 +84,10 @@ type Controller struct {
 }
 
 // RulesKind defines the config kind name of mixer rules.
-const RulesKind = "mixer-rule"
+const RulesKind = "rule"
```

Comment:
`rule` seems fine. does it collide with anything else in istio?

Reply:
There is `routeule`, so I think it is ok to claim `rule`